packaging/ubuntu-16.04/control

```diff
@@ -109,7 +108,8 @@ Description: Support executable to apply confinement for snappy apps
 
 Package: ubuntu-core-launcher
 Architecture: any
-Depends: snap-confine (= ${binary:Version}), ${misc:Depends}, ${shlibs:Depends}
+Depends: snap-confine (= ${binary:Version}), ${misc:Depends}
+Section: oldlibs
 Pre-Depends: dpkg (>= 1.15.7.2)
-Description: Launcher for ubuntu-core (snappy) apps
- This package contains the launcher for launching snappy applications
+Description: transitional dummy package
```

Comment:
When you need to touch this for other reasons, the standard is to capitalize the first word I think.

No need to touch the PR again just for that, though.